.github/workflows/publish-hotfix.yml

```diff
@@ -22,14 +22,16 @@ jobs:
         with:
           fetch-depth: 0 # Depth 0 is required for branch-based versioning
 
-      - uses: actions/setup-node@v4
+      - name: Test code and Create Test Coverage Reports
+        uses: actions/setup-node@v4
         with:
           node-version: 20
           cache: npm
-
-      # Ensure tests pass. Add more checks if required
-      - run: npm ci
-      - run: npm test
+        run: |
+          npm ci
+          npm run format:check
+          npm run lint
+          npm test
```

Comment:
Does the step still fail if e.g the lint fails but the test passes?

Reply:
If the lint fails then this step fails. It won't get to `npm test`

Reply:
Ok just wondered as it was wrapped as a one command step